drivers/storage/scaleio/tests/test-env-up.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+# This script launches infrastructure required for testing the ScaleIO storage driver.
+# It spins up VPC and EC2 instance so AWS account owner will get charged for time
+# that resources will be running.
+
+set -e
+# set -x
+
+AWS_ACCESS_KEY="$1"
+AWS_SECRET_KEY="$2"
+CF_STACK_NAME="$3"
+LAUNCH_KEY_NAME="$4"
+
+# Make sure that aws cli is installed
+hash aws 2>/dev/null || {
```

Comment:
Hi @dvonthenen,

It's not the job of the Travis build to download and install tools necessary for plug-in-specific tests. That's part the purpose of the `test-env-up.sh` script -- to `curl` or install whatever tools are necessary for standing up the test environment, running the tests, and tearing down the environment.